src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```

Comment:
u need to add a private default constructor according to the common mistakes doc
https://mate-academy.github.io/jv-program-common-mistakes/hibernate/configuration/configuration_checklist#:~:text=Add%20a%20private%20default%20constructor%20to%20HibernateUtil%20class%20in%20order%20to%20prevent%20creating%20HibernateUtil%20objects.